### ISSUE TITLE
Tokens Keep Copying And Pasting In Tokens File

### ISSUE THREAD

**Issue by Echoooooooooooooooooooooooooooo** [FuckingToasters/discord-phone-verifier#3]
When the program is ran the tokens keep copying and pasting over and over.

**Comment by FuckingToasters** [comment]
Hmm. Could you send me something which allows me to reproduce it? Like a gif etc.?

**Comment by Echoooooooooooooooooooooooooooo** [comment]
> Hmm. Could you send me something which allows me to reproduce it? Like a gif etc.?

Yes here is a video of it also when i ran this earlier with 5 threads and with like only 10 tokens it copy and pasted them and made 14k of the same proxies. Also another thing it does is after it gets done saving the verified tokens and it saves the next one it deletes the previous one it verified.

https://user-images.githubusercontent.com/107091043/175476788-f7d6e4b2-6b98-40f5-a0f9-9d37f64b5339.mp4

**Comment by FuckingToasters** [comment]
> 

Hmm this is weird. currently each thread is equal to1 token which means that after 1 thread has completed the verification it'll stop and not rerun the script. if you want that if 1 thread is done and then want the same thread to run the script again you need to take a look in main.py on line 158 and add following `verify(proxy_type=proxy_type)` below the line `pystyle.Write.Print("\t[+] Successfully verified Account by Phone!\n", pystyle.Colors.green, interval=0)`

It would be great, if you could provide me with non phone verified tokens (exact amounts you've used) in discord dms: Infinimonster#1312 so i'm able to debug it which allws me to upload a updated version to github.

**Comment by Echoooooooooooooooooooooooooooo** [comment]
> > 
> 
> Hmm this is weird. currently each thread is equal to1 token which means that after 1 thread has completed the verification it'll stop and not rerun the script. if you want that if 1 thread is done and then want the same thread to run the script again you need to take a look in main.py on line 158 and add following `verify(proxy_type=proxy_type)` below the line `pystyle.Write.Print("\t[+] Successfully verified Account by Phone!\n", pystyle.Colors.green, interval=0)`
> 
> It would be great, if you could provide me with non phone verified tokens (exact amounts you've used) in discord dms: Infinimonster#1312 so i'm able to debug it which allws me to upload a updated version to github.

Ok i sent you a friend request

**Comment by FuckingToasters** [comment]
issue has been solved. In case you need help with something else, please open a new issue.
